dashboard/config/foorm/forms/surveys/teachers/teacher_end_of_year_survey.0.json

```diff
@@ -254,19 +250,19 @@
         },
         {
          "value": "unit_2",
-         "text": "Unit 2"
+         "text": "Unit 2 - Digital Information"
         },
         {
          "value": "unit_3",
-         "text": "Unit 3"
+         "text": "Unit 3 - Intro to Programming"
         },
         {
          "value": "unit_4",
-         "text": "Unit 4"
+         "text": "Unit 4 - Big Data and Privacy"
         },
         {
          "value": "unit_5",
-         "text": "Unit 5"
+         "text": "Unit 5 - Building Apps"
         },
         {
          "value": "explore_pt_prep",
```

Comment:
super-nit: there are inconsistent `-` in the last two options: `Explore-PT Prep` and `Create PT Prep`